.github/workflows/continuous-integration.yml

```
@@ -105,6 +105,23 @@ jobs:
           git config --global --add safe.directory "/var/www/glpi/plugins/${{ inputs.plugin-key }}"
           echo -e "\033[0;33mExecuting Composer install...\033[0m"
           composer install --ansi --no-interaction --no-progress --prefer-dist
+      - name: "Check for common dependencies"
+        if: ${{ !cancelled() && hashFiles(format('{0}/composer.json', inputs.plugin-key)) != '' }}
+        run: |
+          echo -e "\033[0;33mChecking for common dependencies between plugin and core...\033[0m"
+          PLUGIN_DEPS=$(composer show --direct --format=json 2>/dev/null | jq -r '.installed[] | .name' | grep -v '^glpi-project/tools$')
+          CORE_DEPS=$(cd ../../ && composer show --direct --format=json 2>/dev/null | jq -r '.installed[] | .name')
```

I think we should let the stderr be displayed on CI results.

```suggestion
          PLUGIN_DEPS=$(composer show --direct --format=json | jq -r '.installed[] | .name' | grep -v '^glpi-project/tools$')
          CORE_DEPS=$(cd ../../ && composer show --direct --format=json | jq -r '.installed[] | .name')
```